scripts/fetch-pr-comments.mjs

```
@@ -0,0 +1,185 @@
+		const threads = result.repository.pullRequest.reviewThreads.nodes;
+		const unresolvedThreads = threads.filter(thread => !thread.isResolved);
+
+		if (unresolvedThreads.length === 0) {
+			console.log('Открытых комментариев не найдено!');
+			const emptyMarkdown = `# Задачи по ревью PR - #${pullNumber}\n\n`;
+			fs.writeFileSync(outputPath, emptyMarkdown + '✅ Все комментарии закрыты!\n');
+			return;
+		}
+
+		let markdown = `# Задачи по ревью PR - #${pullNumber}\n\n`;
+		markdown += `**Источник:** [PR #${pullNumber} на GitHub](https://github.com/${owner}/${repo}/pull/${pullNumber})\n`;
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)